importer/balanced/builder.go

```diff
@@ -131,7 +132,7 @@ import (
 //
 func Layout(db *h.DagBuilderHelper) (ipld.Node, error) {
 	if !db.IsMultiDagBuilder() {
-		return layout(db)
+		return layout(db, db.IsThereMetaData() && !db.IsReedSolomon())
```

Comment:
Why is it not supporting reed solomon + metadata?

Reply:
Also, dagbuilder should not understand reed solomon. All it understands is that it is a "multi dag builder". So it should not explicitly set a reed solomon flag. Instead, use `db.IsMultiDagBuilder` to check if the reed solomon-like dag is used.

Reply:
My assumption is when an individual splitter instance is running, db.IsMultiDagBuilder() will return false. But this flag will go away after BTFS-931.

Reply:
Got it.